HelloPullRequest/JesperAndreas.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HelloPullRequest
+{
+    class JesperAndreas : Person
+    {
+        public override void Speak()
+        {
+            Console.WriteLine("YEAH PICkLE RICK");
```

Comment:
Reference is both out of date and certainly uncool - recommend more modern memes or mature code.